.gitignore

```diff
@@ -14,4 +14,10 @@ coverage
 *.env
 .DS_Store
 packages/testkit/test-results
-packages/testkit/reports
\ No newline at end of file
+packages/testkit/reports
+
+# Bundle size analysis files
+base.json
+pr.json
+bundle-sizes.md
+.size-limit.js
```

Comment:
maybe put them in a folder in the script and ignore the entire folder?
/bundle-comparison or something?

just in case in the future if you add files to the script they won't sneak into the bundled package in npm